fairing/notebook_helper.py

```diff
@@ -19,9 +18,9 @@ def get_notebook_name():
         for nn in json.loads(response.text):
             if nn['kernel']['id'] == kernel_id:
                 full_path = nn['notebook']['path']
-                return os.path.basename(full_path)
+                return full_path
     
-    return f
+    return None
```

Comment:
Remove the `return` statement altogether?

Reply:
Yup, I don't think I'm using notebook_helper.get_notebook_name, will revert to master